### PR TITLE
feat(config): Add Theme UI monorepo

### PR DESCRIPTION
**Pull request by aaronadamsCA**
## Changes:

Adds the Theme UI monorepo group.

## Context:

I'm using this grouping locally, and wanted to contribute it back to the project.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/config/presets/internal/monorepo.ts [hunk 1]
```diff
@@ -45,6 +45,7 @@ const repoGroups = {
   'System.IO.Abstractions':
     'https://github.com/System-IO-Abstractions/System.IO.Abstractions/',
   'telus-tds': 'https://github.com/telusdigital/tds',
+  'theme-ui': 'https://github.com/system-ui/theme-ui',
```
Comment (viceice) [inline lib/config/presets/internal/monorepo.ts:48]:
```suggestion
  'Theme UI': 'https://github.com/system-ui/theme-ui',
```

fix naming, renovate will slugify for branches as `theme-ui` internally

**Review by rarkins — DISMISSED** [review]

**Comment by rarkins** [comment]
@aaronadamsCA needs updating from main branch

File: lib/config/presets/internal/monorepo.ts [hunk 2]
```diff
@@ -45,6 +45,7 @@ const repoGroups = {
   'System.IO.Abstractions':
     'https://github.com/System-IO-Abstractions/System.IO.Abstractions/',
   'telus-tds': 'https://github.com/telusdigital/tds',
+  'Theme UI': 'https://github.com/system-ui/theme-ui',
```
Comment (viceice) [inline lib/config/presets/internal/monorepo.ts:48]:
```suggestion
  'theme-ui': 'https://github.com/system-ui/theme-ui',
```
@rarkins i think we agreed to only use lowercase without spaces, so i suggest the original version here

**Review by viceice — CHANGES_REQUESTED** [review]

File: lib/config/presets/internal/monorepo.ts (hunk 2 above)
Comment (rarkins) [reply lib/config/presets/internal/monorepo.ts:48]: I'm actually not sure about that. First of all, spaces in the monorepo *name* will be converted to hyphen in the branch name, so it's the same result there. What is the result of this other than that the name of the monorepo in the PR title looks less recognizable?

Comment (viceice) [reply lib/config/presets/internal/monorepo.ts:48]: This is also the preset name: `monorepo:Theme UI`.

So i would now prefer to use the lowercase no space variant.

**Review by viceice — APPROVED** [review]

**Pull request closed without merging**